spec/features/support/mock_zendesk_client.rb

```diff
@@ -0,0 +1,47 @@
+shared_context 'with a mocked support tickets client' do
+  class ZendeskClientMock
```

Comment:
More like a spy this?  Interesting though, I like it

Reply:
After talking with @craigjbass & @danielburnley yesterday I actually don't like it! Can explain more in-person, think I had a level-up moment on how to do this stuff yesterday.

Does the job for now though, was really helpful when growing the thing.